pkg/plugin/settings.go

```diff
@@ -13,24 +13,29 @@ import (
 
 // Settings - data loaded from grafana settings database
 type Settings struct {
-	Server             string `json:"server,omitempty"`
-	Port               int64  `json:"port,omitempty"`
-	Path               string `json:"path,omitempty"`
-	Username           string `json:"username,omitempty"`
-	DefaultDatabase    string `json:"defaultDatabase,omitempty"`
-	InsecureSkipVerify bool   `json:"tlsSkipVerify,omitempty"`
-	TlsClientAuth      bool   `json:"tlsAuth,omitempty"`
-	TlsAuthWithCACert  bool   `json:"tlsAuthWithCACert,omitempty"`
-	Password           string `json:"-,omitempty"`
-	TlsCACert          string
+	Host     string `json:"host,omitempty"`
```

Comment:
Since it's a major update, it's ok, but do you think we can avoid this breaking change?

Reply:
Or we introduce both old and new fields on backward compatibility.

Reply:
I've updated it so that the struct has only the new fields, but `LoadSettings()` will try to read from the old JSON fields first. If the new fields are present they will be overwritten:

```go
if jsonData["server"] != nil {
	settings.Host = jsonData["server"].(string)
}
if jsonData["host"] != nil {
	settings.Host = jsonData["host"].(string)
}
```

Reply:
What is the purpose of settings renaming? It creates additional friction for the migration, so we should challenge every breaking change.